packages/cloudflare/src/utils/image-config.ts

```diff
@@ -0,0 +1,36 @@
+import type { AstroIntegrationLogger } from 'astro';
+import { passthroughImageService, sharpImageService } from 'astro/config';
+
+export function prepareImageConfig(
+	service: string,
+	// FIXME: type
+	config: any,
+	// FIXME: type
+	command: any,
```

Comment:
I would like to get the subtypes, e.g. `AstroConfig['image']`, however that does not seem to work, any ideas how I can get the types?

Reply:
Couldnt you import the type?

Reply:
I can use the type `AstroConfig`, but how to I only get the type of `<config>.image`, since `config` is just the image specific configuration

Reply:
That would be...
```suggestion
	config: AstroConfig["image"],
	// FIXME: type
	command: any,
```

Reply:
I swear I've tried it and it didn't work. Now it works... I love those strange behaviours.... NOT!